clients/pkg/promtail/targets/cloudflare/metrics.go

```diff
@@ -0,0 +1,38 @@
+package cloudflare
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// Metrics holds a set of cloudflare metrics.
+type Metrics struct {
+	reg prometheus.Registerer
+
+	Entries prometheus.Counter
+	LastEnd prometheus.Gauge
+}
+
+// NewMetrics creates a new set of cloudflare metrics. If reg is non-nil, the
+// metrics will be registered.
+func NewMetrics(reg prometheus.Registerer) *Metrics {
+	var m Metrics
+	m.reg = reg
+
+	m.Entries = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "promtail",
+		Name:      "cloudflare_target_entries_total",
+		Help:      "Total number of successful entries sent via the cloudflare target",
+	})
+	m.LastEnd = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "promtail",
+		Name:      "cloudflare_target_last_end_timestamp",
```

Comment:
`<nit>`Elsewhere we call a metric like this `highest_timestamp` (like in remote-write).`</nit>`

Reply:
I can make that change.